core/profiles/standard/standard.install

```diff
@@ -169,53 +191,10 @@ function standard_install() {
   config('path.settings')
     ->set('node_page_pattern', '[node:title]')
     ->set('node_post_pattern', 'posts/[node:title]')
+    ->set('node_card_pattern', 'cards/[node:title]')
```

Comment:
We could remove the path alias pattern, since these nodes will never be seen at their paths.